po/zh_CN.po

```diff
@@ -0,0 +1,1315 @@
+#. * hours, 5 minutes, 2 seconds and 50 milliseconds. You may change ":" to
+#. * the separator that your locale uses or use "%Id" instead
+#. * of "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:93
+#, c-format
+msgctxt "long time format"
+msgid "%d:%02d:%02d.%03d"
+msgstr ""
+
+#. -hour:minutes:seconds
+#. Translators: This is a time format, like "-9:05:02" for 9
+#. * hours, 5 minutes, and 2 seconds playback remaining. You may
+#. * change ":" to the separator that your locale uses or use
+#. * "%Id" instead of "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:103
+#, c-format
+msgctxt "long time format"
+msgid "-%d:%02d:%02d"
+msgstr ""
+
+#. -hour:minutes:seconds.msecs
+#. Translators: This is a time format, like "-9:05:02.050" for 9
+#. * hours, 5 minutes, 2 seconds and 50 milliseconds playback remaining. You may
+#. * change ":" to the separator that your locale uses or use
+#. * "%Id" instead of "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:111
+#, c-format
+msgctxt "long time format"
+msgid "-%d:%02d:%02d.%03d"
+msgstr ""
+
+#. -minutes:seconds
+#. Translators: This is a time format, like "-5:02" for 5
+#. * minutes and 2 seconds playback remaining. You may change
+#. * ":" to the separator that your locale uses or use "%Id"
+#. * instead of "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:124
+#, c-format
+msgctxt "short time format"
+msgid "-%d:%02d"
+msgstr ""
+
+#. -minutes:seconds.msec
+#. Translators: This is a time format, like "-5:02.050" for 5
+#. * minutes 2 seconds and 50 milliseconds playback remaining. You may change
+#. * ":" to the separator that your locale uses or use "%Id"
+#. * instead of "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:132
+#, c-format
+msgctxt "short time format"
+msgid "-%d:%02d.%03d"
+msgstr ""
+
+#. minutes:seconds.msec
+#. Translators: This is a time format, like "5:02" for 5
+#. * minutes 2 seconds and 50 milliseconds. You may change ":" to the
+#. * separator that your locale uses or use "%Id" instead of
+#. * "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:143
+#, c-format
+msgctxt "short time format"
+msgid "%d:%02d.%03d"
+msgstr ""
+
+#. minutes:seconds
+#. Translators: This is a time format, like "5:02" for 5
+#. * minutes and 2 seconds. You may change ":" to the
+#. * separator that your locale uses or use "%Id" instead of
+#. * "%d" if your locale uses localized digits.
+#.
+#: src/libvalent/ui/valent-time-helpers.c:152
+#, c-format
+msgctxt "short time format"
+msgid "%d:%02d"
+msgstr ""
+
+#: src/libvalent/ui/valent-window.c:186
+msgid "Unpaired"
+msgstr "未配对"
+
+#: src/libvalent/ui/valent-window.c:196
+msgid "Connected"
+msgstr "已连接"
+
+#: src/libvalent/ui/valent-window.c:345
+msgid "translator-credits"
+msgstr ""
```

Comment:
Sorry, I should have checked this! If you want you can translate this to your name, and it will show up in the "About" dialog automatically :slightly_smiling_face:

Reply:
Sure, I missed it. I will add it in the next pr. For I believe some of my translations may need to be polished again.